docs/user_guide/secure_payloads.rst

```diff
@@ -0,0 +1,93 @@
+Secure Payloads
```

Comment:
Sometimes the term "secure payloads" is used and sometimes "trusted payloads". Should we not try to harmonize the terminology if there is no difference between secure and trusted  ?

Reply:
i prefer secure payloads, since they are in fact encrypted.  They are trusted too, but that's less important to most people

Reply:
@nabilschear I trust you ;-)
So is your proposal to always use "secure payloads" throughout the documentation and no more "trusted payloads" ?

Reply:
Changed all to Secure Payloads. I think this is better, trust does not come into it (as you're the one sending the payload. so it figures you would hopefully trust yourself), it is however secure, in that enrolment happens first and then keys are exchanged.